src/harness/fake_sas.py

```diff
@@ -182,6 +182,16 @@ def Deregistration(self, request, ssl_cert=None, ssl_key=None):
         })
     return response
 
+  def GetSasImplementationRecord(self, request, ssl_cert=None, ssl_key=None):
```

Comment:
> Regarding fake_sas.py changes:
> GetSasImplementationRecord method is not overrided in fakeSAS handler in master version but it was defined in sas_interface.py
> that cause fake_sas stopped working and not able to register devices. So we have fixed this defect in fake_sas.py in order to test GRA_5 and GRA_6


You're right, that method was taken out of the fake SAS in https://github.com/Wireless-Innovation-Forum/Spectrum-Access-System/pull/458 . It's my fault that we didn't also remove it from sas_interface.py and sas.py, which is why you're running into problems. We'll issue a PR to fix this ASAP, and then I think you should be able to revert the changes in this file.

Reply:
Ok, we have removed this fix in PR440